### ISSUE TITLE
Problem with "Window Title" feature and Overwatch

### ISSUE THREAD

**Issue by TonyDong51089** [WarmUpTill/SceneSwitcher#329]
Window title function doesn't work properly after I play overwatch. I don't have any settings specifically for overwatch. Anything I need to provide?[](ur
![Screenshot 2021-10-21 222059](https://user-images.githubusercontent.com/19286058/138299795-2f57367a-609d-4c61-b752-d10022087ca1.png)
![Screenshot 2021-10-21 222600](https://user-images.githubusercontent.com/19286058/138299874-f870c943-2c7c-4144-a2d9-ca94f3eea5eb.png)
l)

**Comment by TonyDong51089** [comment]
Currently I cannot reproduce that unexpected behavior, and probably isn't Overwatch's fault.
I have a regular expression in Window title function where "when the window has the word 'YouTube' in it, it will switch scene, if doens't, it will not switch scene." The expression is ".*YouTube.*"
It works most of the time, whereas after I played Overwatch, it doesn't seems to be responding. I close and open chrome (where "youtube" will only appear), and it functions again.
I have only tuned "program (or application, idk I have chinese version here)" and "window title" function, others are left default. I can guarantee I do not have a specific setting for Overwatch
